test/integration/golden_response/staging/get_triples/place_type.json

```diff
@@ -4945,7 +4945,7 @@
       "objectTypes": [
         "Class"
       ],
-      "provenanceId": "dc/sm3m2w3"
+      "provenanceId": "dc/5n63hr1"
```

Comment:
would be nice to find a way to stabilize the provenance..

Reply:
Yeah, that is a long-standing TODO... I'll look into it this week.